src/components/header-block/header-block.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+import { Illustration, IllustrationSize } from '../illustration/illustration';
+import { IllustrationType } from '../illustration/illustration-set';
+import { down, up } from '../breakpoint/breakpoint';
+import { getMainProps } from 'gatsby-plugin-image/dist/src/components/hooks';
+
+interface HeaderBlockProps {
+  topline: string;
+  headline: string;
+  metaline: string;
+  children: string;
+  large: boolean;
+  illustration?: IllustrationType | null;
+}
+
+interface LargeProps {
+  large: boolean;
+}
+
+const TopLineR = css`
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 110%;
+`;
+
+const HeadlineXL = css`
+  font-weight: bold;
+  font-size: 48px;
+  line-height: 110%;
+`;
+
+const TextS = css`
+  font-size: 14px;
+  line-height: 150%;
+`;
+
+const TextL = css`
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 150%;
+`;
+
+const Text = css`
```

Comment:
If I remember correctly those are now all defined in `typography-v2` right? So can should be able to update the branch and remove them here.